Source/OEXVideoSummary.m

```diff
@@ -171,6 +171,9 @@ - (BOOL) isSupportedVideo {
         if (([encoding URL] && [OEXInterface isURLForVideo:[encoding URL]]) && [self isSupportedEncoding:name]) {
             isSupportedEncoding = true;
             break;
+        }else if ([[encoding name] isEqualToString:OEXVideoEncodingYoutube] && OEXConfig.sharedConfig.youtubeVideoConfig.enabled) {
```

Comment:
duplicated code; break the if or save the booleans into variables to avoid code duplication (also it says `}else if` instead of `} else if {`)